master/internal/api_experiment.go

```diff
@@ -43,3 +46,54 @@ func (a *apiServer) GetExperiments(
 	a.sort(resp.Experiments, req.OrderBy, req.SortBy, apiv1.GetExperimentsRequest_SORT_BY_ID)
 	return resp, a.paginate(&resp.Pagination, &resp.Experiments, req.Offset, req.Limit)
 }
+
+// For the 4 following endpoints: activate, pause, cancel, and kill if the experiment is not in
```

Comment:
blocking: on second thought, we can quickly query for the id before the send the request and then return a 404 if it does not exist.

Reply:
separate PR ?

Reply:
I added this to this PR anyway

Reply:
![Screen Capture_select-area_20200708113640](https://user-images.githubusercontent.com/12127420/86957255-4edd0800-c10f-11ea-8554-e0c28bc3cc0f.png)
a github bug?